rholang/src/main/scala/rholang/interpreter/Normalize.scala

```diff
@@ -22,18 +22,58 @@ trait GroundNormalizeVisitor extends AbsynGround.Visitor[Ground, Any]
   override def visit( gi: GroundUri, n: Any ) : Ground = GUri(gi.uri_)
 }
 
+trait NameNormalizeVisitor extends Name.Visitor[NameVisitOutputs, NameVisitInputs] {
```

Comment:
It's generally customary to put classes or traits in their own separate file.

See:
https://docs.scala-lang.org/style/files.html

If you feel like this falls under the "Multi-Unit File" provision, then you should probably use a lower camel-case name, as the article suggests.

Reply:
I'm going to lowercase the filename and leave it for now.